app/src/pages/ProfileEdit.tsx

```diff
@@ -0,0 +1,126 @@
+import * as React from "react"
+import QuestionField from "../components/ProfileEdit/QuestionField"
+
+import styled from "styled-components"
+import { Button, LinearProgress } from '@material-ui/core';
+import { Question } from "../models/Question"
+
+const Container = styled.div`
+  margin: 30px auto;
+  padding: 0 15px;
+  max-width: 1140px;
+`
+
+const StyledButton = styled(Button)`
```

Comment:
looks like we are rebuilding a button in a lot of place, let's at some point think about button reusability! we can build a generic button that take an onclick handler as a prop and passes in a callback. and reuse it throughout the app.

nothing to change here - throwing it out for us to think about with this and other components we reuse often.